.golangci.yml

```diff
@@ -1,19 +1,18 @@
 linters:
   enable:
-    - structcheck
-    - varcheck
-    - staticcheck
-    - unconvert
+    - exportloopref # Checks for pointers to enclosing loop variables
     - gofmt
     - goimports
-    - revive
+    - gosec
     - ineffassign
-    - vet
-    - unused
     - misspell
-    - gosec
-    - exportloopref # Checks for pointers to enclosing loop variables
+    - nolintlint
```

Comment:
:-)

Reply:
Yeah, it's a horrible name 😂 

We need to look how well this one works; in some cases it may need a bit of hand-holding if things are unused by "some" platforms/arches, but actually used in others. That said, I think "explicit > implicit", so having to add a `nolintlint` comment to indicate "ACK: this is ok to ignore" may be worth it if it helps cleaning up actual unused `//nolint` comments.